CHANGELOG.md

```diff
@@ -1,3 +1,83 @@
+### citus v9.0.0 (October 7, 2019) ###
+
+* Adds support for PostgreSQL 12
+
+* Adds UDFs to help with PostgreSQL upgrades
+
+* Delegates `SELECT function()` to a worker node
+
+* Distributes types to worker nodes
+
+* Introduces `create_distributed_function` UDF
+
+* Introduces local query execution for Citus MX
+
+* Implements infrastructure for routing `CALL` to MX workers
+
+* Adds a feature flag to turn off `CREATE TYPE` propagation
+
+* Adds dist. arg. and colocation parameters to `create_distributed_function()`
```

Comment:
I think we don't need separate items for create_distributed_function given that it is introduced newly.

Reply:
Makes sense, I removed 2 small items about `create_distributed_function`